handlers/deploy.go

```diff
@@ -22,6 +23,12 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
+// WatchdogPort for the OpenFaaS function watchdog
+const WatchdogPort = 8080
+
+// InitialReplicas how many replicas to start of creating for a function
+const InitialReplicas = 1
```

Comment:
Worth allowing the user to set the initial/min number of replicas when deploying a function?

Also does this actually need to be exported? its only used in this file.

Reply:
Number 1 is out of scope for PR, but can come under the scope of the issue in the faas repo around setting max replicas.